Menu Entidades agregado

loranca opened the pull request.

loranca committed:
Menu Entidades agregado

hbkruzpe committed:
agregar item proyectos

mwleinad commented on resources/js/components/NavigationComponent.vue:
```diff
@@ -24,6 +24,14 @@
             </v-list-item>
+            <v-list-item link :to="{ name: 'entidades' }">
+                <v-list-item-action>
+                    <v-icon>mdi-contact-mail</v-icon>
```
Supongo que este no es el icono correcto?

loranca committed:
Se resolvio el conflicto
Merge remote-tracking branch 'origin/hb' into avance1

# Conflicts:
#	resources/js/components/NavigationComponent.vue
